ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -55,41 +55,42 @@ class IbmMqCheck(AgentCheck):
 
     CHANNEL_COUNT_CHECK = 'ibm_mq.channel.count'
 
-    def check(self, instance):
-        config = IBMMQConfig(instance)
-        config.check_properly_configured()
+    def __init__(self, name, init_config, instances):
+        super(IbmMqCheck, self).__init__(name, init_config, instances)
```

Comment:
Let's not enforce the signature here? Also this is missing the constructor signature so `self.config` won't be type-checked. Suggestion:

```suggestion
    def __init__(self, *args, **kwargs):
        # type: (*Any, **Any) -> None
        super(IbmMqCheck, self).__init__(*args, **kwargs)
```